include/hlslib/common/OpenCL.h

```diff
@@ -1106,7 +1149,7 @@ class Buffer {
 #endif
   size_t nElements_;
 
-};  // End class Buffer
+}; // End class Buffer
```

Comment:
```suggestion
};  // End class Buffer
```
Don't change formatting